test/polyfill.test.js

```diff
@@ -315,13 +315,14 @@ describe('Constructible Style Sheets polyfill', () => {
         checkShadowCss(element, {height: '82px'}, {width: '53px'});
       });
 
-      it('forbids assigning a non-Array value to adoptedStyleSheets', async () => {
+      it('forbids assigning a non-Array value to adoptedStyleSheets', async (done) => {
```

Comment:
`done` is unnecessary for async tests, it is a workaround for old node callback approach.

Reply:
That's strange. Does it appear each time? Does it affect all browsers? I ran tests for this branch on my Win10 with headless Firefox and Chrome, and it works as expected, no failing.